lib/features/family_goal_tracker/widgets/goal_active_tracker.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app_kids/features/family_goal_tracker/cubit/goal_tracker_cubit.dart';
+import 'package:givt_app_kids/features/family_goal_tracker/widgets/gradient_progress_bar.dart';
+import 'package:givt_app_kids/helpers/app_theme.dart';
+
+class GoalActiveWidget extends StatelessWidget {
```

Comment:
a tiny ask to try keeping same names for class and filename) the reason - I think it's easier to search for/ navigate to such files..